neat_core.c

```diff
@@ -486,14 +649,14 @@ static int io_readable(neat_ctx *ctx, neat_flow *flow,
     struct sctp_recvv_rn rn;
     socklen_t infolen = sizeof(struct sctp_recvv_rn);
     int flags = 0;
-#endif
-#endif
+#endif // else !defined(USRSCTP_SUPPORT)
+#endif // defined(SCTP_SUPPORTED)
     neat_log(NEAT_LOG_DEBUG, "%s", __func__);
 
     if (!flow->operations || !flow->operations->on_readable) {
         return READ_WITH_ERROR;
     }
-#if defined(IPPROTO_SCTP)
+#if defined(SCTP_SUPPORTED)
     if ((flow->sockProtocol == IPPROTO_SCTP) &&
```

Comment:
But it would compile. And when you load an NKE to add SCTP support it would work...
The general rule of the #ifdef mess I used was to compile in what is possible to compile in...